app/controllers/topics_controller.rb

```diff
@@ -691,6 +691,8 @@ def invite
       end
     rescue Topic::UserExists => e
```

Comment:
Both of these exceptions return the same thing so you could handle both on one line with a comma.